packages/js/onboarding/src/components/WooPaymentsMethodsLogos/PaymentMethodsLogos.scss

```diff
@@ -38,3 +53,21 @@
 		}
 	}
 }
+
+.inside-popover {
+	display: grid;
+	grid-template-columns: repeat(4, 1fr);
+	grid-template-rows: repeat(3, 1fr);
+	gap: 8px;
```

Comment:
```suggestion
	gap: $gap-smaller;
```

Reply:
Oh thanks! I used `gap-small` and it was too much, but I didn't think about `smaller` 😅 

I fixed all occurences